src/targets/gpu/compile_hip_code_object.cpp

```diff
@@ -177,7 +178,7 @@ operation compile_hip_code_object(const std::string& content, hip_compile_option
     options.params += " " + join_strings(compiler_warnings(), " ");
     options.params += " -ftemplate-backtrace-limit=0";
     options.params += " -Werror";
-    auto cos = compile_hip_src(srcs, std::move(options.params), get_device_name());
+    auto cos = compile_hip_src(srcs, options.params, get_device_name());
```

Comment:
Why is `std::move` being removed?

Reply:
Put back `std::move`.  It was the remainder from earlier changeset .